tests/base_case.py

```diff
@@ -70,4 +70,3 @@ def setUp(self):
         self.chatbot.train(data1)
         self.chatbot.train(data2)
         self.chatbot.train(data3)
-
```

Comment:
Not a major issue, but PEP8 does specify [a blank](http://pep8.readthedocs.org/en/latest/intro.html) line at the end of files.